apps/admin/backend/src/util/get_current_time.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Simply a wrapper for `Date.now()`, returns the current time in Unix time.
+ *
+ * Separated out into its own file so that it can be mocked in tests.
+ */
+export function getCurrentTime(): number {
```

Comment:
It never occurred to me! Since this was made for reports, and reports just take the `number` milliseconds since format, that's how I set up the helper. If you think it'd be worth it, could follow up so that the helper and the reports all use `luxon` for immutability.